projects/core/src/occ/adapters/user/default-occ-user-config.ts

```diff
@@ -7,17 +7,13 @@ export const defaultOccUserConfig: OccConfig = {
         /* eslint-disable max-len */
         paymentDetailsAll: 'users/${userId}/paymentdetails',
         paymentDetail: 'users/${userId}/paymentdetails/${paymentDetailId}',
-        orderHistory: 'users/${userId}/orders',
-        orderDetail: 'users/${userId}/orders/${orderId}?fields=FULL',
```

Comment:
I think we should deprecate these instead of deleting them

Reply:
It is not in `public_api`.

Reply:
It might not be in the public api, but I think it will affect functionality if you remove these: the related order adapters will break.